mcs/class/System.Data/Makefile

```diff
@@ -25,6 +25,10 @@ ifdef MONO_FEATURE_APPLETLS
 LIB_MCS_FLAGS += -d:MONO_FEATURE_APPLETLS
 endif
 
+ifdef ENABLE_GSS
+LIB_MCS_FLAGS += -d:ENABLE_GSS
```

Comment:
What is this for?

Reply:
if GSS is not enabled then we do not include [DllImport] and use stubs (like it was before). As far as I remember "dead" DllImports cause problems for AOT, see https://github.com/mono/corefx/blob/master/src/Common/src/Interop/Unix/System.Net.Security.Native/Interop.NetSecurityNative.cs#L15

Reply:
ENABLE_GSS won't be set for BCL builds, you need to do it manually via `mcs/build/config.make`